modules/seqexec/server/src/main/scala/seqexec/server/ObserveEnvironment.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server
+
+import cats.data.Reader
+import gem.Observation
+import edu.gemini.spModel.config2.Config
+import seqexec.server.keywords._
+
+/**
+  * Describes the parameters for an observation
+  */
+final case class ObserveEnvironment[F[_]](
```

Comment:
We need a bunch of parameters to setup an obs. Passing them in a data class makes the signatures simpler